document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/service/impl/AccountsServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.document.generator.accounts.service.impl;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+import uk.gov.companieshouse.document.generator.accounts.data.accounts.AccountsManager;
+import uk.gov.companieshouse.document.generator.accounts.service.AccountsService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Service
+public class AccountsServiceImpl implements AccountsService {
```

Comment:
Shouldn't this be called AbridgedAccountsServiceImpl? The JavaDoc on the method which calls this class says "Gets the abridged account data from abridged specific services". The data in each account type is different and the endpoints could be different, they should be in different implementations?

Reply:
Sorry, this is the correct name. The JD is also correct. I guess this is the downside of copy and pasting an already written solution just to make PRs smaller. They are both correct, however the next PR I will insert the abridged implementation. Which is where it will all make sense.